timemachine/fe/free_energy.py

```diff
@@ -1216,18 +1221,30 @@ def get_swap_acceptance_rates(fraction_accepted_by_pair):
             instantaneous_swap_acceptance_rates = get_swap_acceptance_rates(fraction_accepted_by_pair)
             average_swap_acceptance_rates = get_swap_acceptance_rates(np.sum(fraction_accepted_by_pair_by_iter, axis=0))
 
+            wall_time_per_frame_current = (current_time - last_update_time) / print_diagnostics_interval
+            wall_time_per_frame_average = (current_time - begin_loop_time) / (current_frame + 1)
+            estimated_wall_time_remaining = wall_time_per_frame_average * (md_params.n_frames - (current_frame + 1))
+
+            def format_wall_time(time_seconds):
+                return f"{time_seconds:.2f} s"
+
             def format_rate(r):
                 return f"{r * 100.0:5.1f}%"
 
             def format_rates(rs):
                 return " | ".join(format_rate(r) for r in rs)
 
-            print("Frame ", current_frame + 1)
-            print("Acceptance rates (inst.)   :", format_rates(instantaneous_swap_acceptance_rates))
-            print("Acceptance rates (average) :", format_rates(average_swap_acceptance_rates))
-            print("Final replica permutation  :", hrex.replica_idx_by_state)
+            print("Frame", current_frame + 1)
+            print("Wall time per frame, current   :", format_wall_time(wall_time_per_frame_current))
```

Comment:
Could we condense these three lines into one? Make it a bit easier to parse and avoids ordering artifacts in logging

Reply:
Could do; I opted to keep a simple `key: value` format here since I find that simplest to write and parse. To be clear, are you suggesting something like:

```
Wall time: 0.12 s per frame, 0.12 s per frame average, 123.45 s remaining
```

Reply:
Yes, something like that, with a slight preference for the remaining time first.

I find it clearer to read when reading large log files, especially if they are sorted by time stamp which can change up the ordering

Reply:
OK, as of https://github.com/proteneer/timemachine/pull/1330/commits/44052403fe0026c94fdbbce2d9a1631c77bc354d, the output now looks like:

```
Frame 20
Approximately 612.04 s remaining at 0.625 s/frame (0.606 s/frame since last message)
HREX acceptance rates, current :   8.9% |  17.1% |  53.2% |  30.9% |   5.7% |  21.8% |   5.9% |   4.7% |  18.6% |  79.9% |  75.0% |  45.7% |  55.6% |  39.6% |  18.7% |  51.8% |  66.7% |  50.9% |  24.3% |  28.8% |  48.1% |  66.5% |  56.9% |  57.7% |  52.7% |  27.4% |  27.2% |  36.9% |  21.7% |  28.7% |  70.9% |  40.5% |  40.3% |  49.7% |  53.6% |  52.7% |   8.0% |  37.0% |  82.5% |  84.5% |  10.5% |  77.5% |  64.7% |  72.3% |  16.4%
HREX acceptance rates, average :  39.5% |  38.0% |  60.8% |  53.3% |  29.2% |  28.2% |  39.1% |  39.4% |  53.3% |  46.8% |  56.9% |  64.3% |  54.2% |  62.5% |  55.8% |  52.3% |  41.8% |  51.5% |  30.0% |  47.1% |  52.5% |  65.6% |  64.3% |  59.6% |  47.3% |  30.4% |  48.4% |  41.2% |  50.2% |  44.5% |  53.2% |  48.8% |  35.6% |  55.1% |  45.9% |  52.6% |  41.9% |  32.1% |  67.5% |  52.6% |  49.3% |  69.8% |  59.4% |  57.0% |  40.1%
HREX replica permutation       : [0, 2, 13, 4, 3, 6, 1, 7, 15, 18, 11, 5, 10, 8, 17, 14, 16, 12, 9, 19, 22, 21, 28, 20, 23, 24, 27, 30, 33, 26, 29, 31, 36, 37, 34, 25, 35, 45, 32, 38, 43, 44, 40, 42, 41, 39]
```